Makefile

```diff
@@ -908,25 +899,19 @@ warn-auto.sh mailsubj.sh conf-qmail conf-break conf-split
 mailsubj.0: \
 mailsubj.1
 
-make-compile: \
-make-compile.sh auto-ccld.sh
-	cat auto-ccld.sh make-compile.sh > make-compile
-	chmod 755 make-compile
-
-make-load: \
-make-load.sh auto-ccld.sh
-	cat auto-ccld.sh make-load.sh > make-load
-	chmod 755 make-load
+compile: \
```

Comment:
Please keep the alphabetical order of the Makefile targets.

Reply:
I am now learning this best practice for large makefiles.